code/LeftAndMain.php

```diff
@@ -316,10 +328,15 @@ public function getCombinedClientConfig()
      */
     public function getClientConfig()
     {
+        // Allows the section name to be overridden in descendant classes
+        if (!$name = $this->config()->get('section_name')) {
```

Comment:
It's pretty minor, but I think I'd prefer to see the assignment on its own line

Reply:
Changed 👍